src/main/java/wolox/training/models/Book.java

```diff
@@ -0,0 +1,156 @@
+package wolox.training.models;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name = "books")
+public class Book {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.SEQUENCE)
+	private int id;
+
+	private String genre;
+
+	@Column(nullable = false)
```

Comment:
It would be better to use the `@NotNull` annotation, so if a user sends a null attribute, your application will fail as soon as possible, without even trying to get to de DB. More info about de annotation and similar constraints [here](https://www.baeldung.com/java-bean-validation-not-null-empty-blank).

Reply:
Thanks a lot for you advice. I thought about that, and implemented it but within the next PR that is the CRUD operations with exception handling.